packages/create-sitecore-jss/scripts/watch-templates.ts

```diff
@@ -49,6 +81,9 @@ const initializeApps = async (noInstall: boolean) => {
     if (watch.args.restoreLockfile) {
```

Comment:
We probably wouldn't need to restore lock file if we are regenerating. Seems like regenerating would take precedence.

Reply:
Right, this scenario I mentioned would only affect react and wouldn't really break anything would just be unnecessary work (i.e. if you had restoreLockFile set when initializing react, it would restore the lock file then delete and regen it).

Reply:
So, I suppose restoreLockFile wouldn't really do much for react then? Which, honestly, I think is OK. Let's leave it as-is :)